src/elements/text-input/package.json

```diff
@@ -15,6 +15,8 @@
     "@emotion/core": "^10.0.10",
     "@uswitch/trustyle.frozen-input": "^0.1.3",
     "@uswitch/trustyle.styles": "^0.1.3",
-    "react": "^16.7.0"
+    "ramda": "^0.26.1",
```

Comment:
Remove this please :)

Reply:
I might have forgotten -S when doing npm uninstall 🤦‍♂